doc/dbus/org.opensuse.Agama1.Network.Connection.IP.doc.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<node>
+  <interface name="org.freedesktop.DBus.Introspectable">
+    <method name="Introspect">
+      <arg type="s" direction="out"/>
+    </method>
+  </interface>
+  <interface name="org.freedesktop.DBus.Peer">
+    <method name="Ping">
+    </method>
+    <method name="GetMachineId">
+      <arg type="s" direction="out"/>
+    </method>
+  </interface>
+  <interface name="org.freedesktop.DBus.Properties">
+    <method name="Get">
+      <arg name="interface_name" type="s" direction="in"/>
+      <arg name="property_name" type="s" direction="in"/>
+      <arg type="v" direction="out"/>
+    </method>
+    <method name="Set">
+      <arg name="interface_name" type="s" direction="in"/>
+      <arg name="property_name" type="s" direction="in"/>
+      <arg name="value" type="v" direction="in"/>
+    </method>
+    <method name="GetAll">
+      <arg name="interface_name" type="s" direction="in"/>
+      <arg type="a{sv}" direction="out"/>
+    </method>
+    <!--
+     Emits the `org.freedesktop.DBus.Properties.PropertiesChanged` signal.
+     -->
+    <signal name="PropertiesChanged">
+      <arg name="interface_name" type="s"/>
+      <arg name="changed_properties" type="a{sv}"/>
+      <arg name="invalidated_properties" type="as"/>
+    </signal>
+  </interface>
+  <interface name="org.opensuse.Agama1.Network.Connection">
+    <!--
+     Connection ID.
+
+     Unique identifier of the network connection. It may or not be the same that the used by the
+     backend. For instance, when using NetworkManager (which is the only supported backend by
+     now), it uses the original ID but appending a number in case the ID is duplicated.
+     -->
+    <property name="Id" type="s" access="read"/>
+    <property name="Interface" type="s" access="readwrite"/>
+  </interface>
```

Comment:
All the above interfaces should be omitted. Each `foo.doc.xml` file should only contain one `<interface>` element, `<interface name="foo">`

That is why https://github.com/openSUSE/agama/actions/runs/6300720786/job/17107419428?pr=769 is failing

Reply:
Thanks!. Now I got it. I have fixed those issues.